src/gui/editorwidgets/qgsvaluemapconfigdlg.cpp

```diff
@@ -48,13 +49,17 @@ QgsEditorWidgetConfig QgsValueMapConfigDlg::config()
     if ( !ki )
       continue;
 
+    QString ks = ki->text();
+    if (( ks == QString( "NULL" ) ) && !( ki->flags() & Qt::ItemIsEditable ) )
+      ks = "{2839923C-8B7D-419E-B84B-CA2FE9B80EC7}";
```

Comment:
can you make this a #define somewhere to avoid the multiple places it's manually reused?